src/fiatconnect/ReviewScreen.tsx

```diff
@@ -34,6 +43,33 @@ export default function FiatConnectReviewScreen({ route, navigation }: Props) {
   const { t } = useTranslation()
   const dispatch = useDispatch()
   const { flow, normalizedQuote, fiatAccount } = route.params
+  const providerId = normalizedQuote.getProviderId()
+
+  const fiatConnectQuotesError = useSelector(fiatConnectQuotesErrorSelector)
+  const fiatConnectQuotesLoading = useSelector(fiatConnectQuotesLoadingSelector)
+  const fiatConnectQuotes = useSelector(fiatConnectQuotesSelector)
+  const [quote, setQuote] = useState(normalizedQuote)
+  const [showingExpiredQuoteDialog, setShowingExpiredQuoteDialog] = useState(false)
+  const [usingUpdatedQuote, setUsingUpdatedQuote] = useState(false)
+
+  useEffect(() => {
+    if (!fiatConnectQuotesLoading && usingUpdatedQuote) {
+      const updatedQuoteData = fiatConnectQuotes.find(
+        (quote) => quote.provider.id === providerId && quote.ok
+      )
+      if (fiatConnectQuotesError || !updatedQuoteData) {
+        dispatch(showError(ErrorMessages.QUOTE_UPDATE_FAILED))
```

Comment:
Once this error is shown, what action can the user take? Click continue and trigger another quote refresh?
Also, can we add a unit test for checking whether this is dispatched on error?